src/pages/nft/class/_classId/index.vue

```diff
@@ -239,13 +239,29 @@ export default {
       });
       return;
     }
-    await Promise.all([
-      store.dispatch('fetchNFTMetadata', classId),
-      store
-        .dispatch('lazyGetNFTPurchaseInfo', classId)
+    try {
+      await Promise.all([
+        store.dispatch('fetchNFTMetadata', classId),
+        store
+          .dispatch('lazyGetNFTPurchaseInfo', classId)
+          // eslint-disable-next-line no-console
+          .catch(err => console.error(err)),
```

Comment:
can we json stringify this error too? log spam on not found

Reply:
Updated